benchmark_utils/__init__.py

```diff
@@ -3,4 +3,11 @@
 # name `benchmark_utils`, and code defined inside will be importable using
 # the usual import syntax
 
-# on peut définir ici des fonctions 
+from benchopt import safe_import_context
+
+# Protect the import with `safe_import_context()`. This allows:
+# - skipping import to speed up autocompletion in CLI.
+# - getting requirements info when all dependencies are not installed.
+with safe_import_context() as import_ctx:
+    from .transformation import smooth_timemask, channels_dropout # noqa F401
```

Comment:
avoid using `noqa` when exposing function in a module.
good practice is to put the exposed function in a `__all__` variable:

`__all__ = ['smooth_time_mask', ...]`

and this works with the linter.

Reply:
Nice, I didn't know this. Thank you!